privilege/privileges/ldap/ldap_common.go

```diff
@@ -0,0 +1,376 @@
+// Copyright 2023-2023 PingCAP Xingchen (Beijing) Technology Co., Ltd.
```

Comment:
@Defined2014 I think it should be removed.

Reply:
Done